charts/ssv-node/templates/service.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ include "ssv-node.fullname" . }}-p2p"
+  labels:
+    {{- include "ssv-node.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  externalTrafficPolicy: Local
+  ports:
+    - port: {{ .Values.service.ports.p2pTCP }}
+      targetPort: p2p-tcp
+      protocol: TCP
+      name: p2p-tcp
+    - port: {{ .Values.service.ports.p2pUDP }}
+      targetPort: p2p-udp
+      protocol: UDP
+      name: p2p-udp
+  selector:
+    {{- include "ssv-node.selectorLabels" . | nindent 4 }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ include "ssv-node.fullname" . }}"
```

Comment:
Do you think this should have a suffix name as well?

Reply:
Or perhaps the p2p one should be the main and this a metrics one.

Reply:
That's a good question, sir. I believe the suffix is not needed for the generic service, I added it to the p2p one because I wanted to be specific about it.